### PR TITLE
Add comments to the relevant code according to golint

### PR DESCRIPTION
**Pull request by ZP-AlwaysWin**
**What type of PR is this?**

> /kind cleanup

**What this PR does / why we need it**:
This fixes golint failures of the following file:

-  pkg/util/netsh/testing/

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```
None
```

### CODE REVIEW

File: pkg/util/netsh/testing/fake.go [hunk 1]
```diff
@@ -22,14 +22,15 @@ import (
...
+//FakeNetsh no-op implementation of netsh Interface
 type FakeNetsh struct {
 }
-
+// NewFake return FakeNetsh no-op implementation of netsh Interface
 func NewFake() *FakeNetsh {
 	return &FakeNetsh{}
 }
 
+// EnsurePortProxyRule ensure portproxy rule. if the rule is error ,return error and error message
```
Comment (cmluciano) [inline pkg/util/netsh/testing/fake.go]: Can we re-use the description from the function being tested [here](https://github.com/kubernetes/kubernetes/blob/master/pkg/util/netsh/netsh.go#L69). 

```suggestion
// EnsurePortProxyRule checks if the specified redirect exists and creates it if not found. When the rule results in an error, return the error and its message.
```

Comment (ZP-AlwaysWin) [reply pkg/util/netsh/testing/fake.go]: hi @cmluciano
Yes,you are right,I will Accept your suggestion。

Thank your very much for your help

**Comment by ZP-AlwaysWin** [comment]
> /ok-to-test
> /lgtm

@dcbw

Thank you very much for your reviews,I remodify the function comment, please help to reviews again！

File: pkg/util/netsh/testing/fake.go [hunk 2]
```diff
@@ -22,14 +22,16 @@ import (
...
+// FakeNetsh no-op implementation of netsh Interface
 type FakeNetsh struct {
 }
 
+// NewFake return fakenetsh no-op implementation of netsh Interface
 func NewFake() *FakeNetsh {
 	return &FakeNetsh{}
 }
 
+// EnsurePortProxyRule checks if the specified redirect exists and creates it if not found. When the rule results in an error, return the error and its message.
```
Comment (BenTheElder) [inline pkg/util/netsh/testing/fake.go]: This implementation does no such thing, it always returns `true, nil`. This commment is misleading.

Comment (ZP-AlwaysWin) [reply pkg/util/netsh/testing/fake.go]: Yes，you are right。
I  took a second look at the semantics of this function。 I have revised it according to your opinion。
Thank you very much for your help.
Best Wishes

Comment (BenTheElder) [reply pkg/util/netsh/testing/fake.go]: thanks, and apologies for the terse comment on this earlier, however I do not see an updated comment here?

**Comment by ZP-AlwaysWin** [comment]
> /assgin @BenTheElder

hi,@BenTheElder

I have Modified the annotation of the function to meet your requirements.
please to help me to reviews code again。

Best Wishes

File: pkg/util/netsh/testing/fake.go (hunk 2 above)
Comment (ZP-AlwaysWin) [reply pkg/util/netsh/testing/fake.go]: hi @BenTheElder 

I modified the function description again；Please see if it's suitable.

Best Wishes

File: pkg/util/netsh/testing/fake.go [hunk 3]
```diff
@@ -22,15 +22,18 @@ import (
...
+// FakeNetsh is a no-op implementation of the netsh Interface
 type FakeNetsh struct {
 }
 
+// NewFake returns a fakenetsh no-op implementation of the netsh Interface
 func NewFake() *FakeNetsh {
 	return &FakeNetsh{}
 }
 
+// EnsurePortProxyRule checks if the specified redirect exists and creates it if not found.Now it always returns true, nil
```
Comment (BenTheElder) [inline pkg/util/netsh/testing/fake.go]: I think we can just say that this is implementing the netsh interface and always returns true without any error? 
Describing what the normal method does and then saying this one doesn't do that seems roundabout /shrug

Comment (ZP-AlwaysWin) [reply pkg/util/netsh/testing/fake.go]: hi @BenTheElder

I modified the description of the function again, please reviews the code again。 

Best Wishes

**Comment by ZP-AlwaysWin** [comment]
/assign @sttts
hi @sttts 

please help me to review the pr

Best Wishes

**Pull request merged**